services/web/server/src/async_sio.py

```diff
@@ -95,7 +95,7 @@ def connect(sid, environ):
         dataOut["url"] = result
         await SIO.emit('presignedUrl', data=dataOut, room=sid)
     except ResponseError as err:
-        print(err)
+        _LOGGER.exception("Failed client %s requests S3 url for %s: %s", sid, data, err)
```

Comment:
There is no need to pass ``err``. See my example and [manual](https://docs.python.org/3/library/logging.html#logging.Logger.exception)

Reply:
Got it, thanks!